au/code/au/quantity_point_test.cc

```diff
@@ -372,6 +372,11 @@ TEST(QuantityPoint, CanCompareUnitsWithDifferentOrigins) {
     EXPECT_THAT(celsius_pt(0), ConsistentlyLessThan(kelvins_pt(274)));
 }
 
+TEST(QuantityPoint, AddingPosUnitQuantityToNegUnitPointGivesPosUnitPoint) {
+    constexpr auto neg_celsius_pt = celsius_pt * (-mag<1>());
+    EXPECT_THAT(neg_celsius_pt(40) + celsius_qty(15), PointEquivalent(celsius_pt(-25)));
```

Comment:
"GivesPosUnitPoint"?  It looks like the unit point we're expecting is negative?

Reply:
The _unit itself_ for the point we're expecting is positive, though: we get `celsius_pt` rather than `neg_celsius_pt`.